modules/thank_you_page/EED_Thank_You_Page.module.php

```diff
@@ -363,7 +370,7 @@ public function init()
         $this->_primary_registrant = $this->_current_txn->primary_registration() instanceof EE_Registration
             ? $this->_current_txn->primary_registration()
             : null;
-        $this->_is_primary = $this->_primary_registrant->reg_url_link() === $this->_reg_url_link ? true : false;
+        $this->_is_primary = $this->_primary_registrant->reg_url_link() === $this->_reg_url_link;
```

Comment:
😄

Reply:
this was my doing... I came from the world of CodeIgniter and Expression Engine and their formatting standard was to be 1000% explicit with things so that it was instantly clear what was being returned.

This case is obvious but sometimes it's not so clear